wagtail/wagtailimages/models.py

```diff
@@ -254,7 +258,9 @@ def get_rendition(self, filter):
             if cache_key:
                 output_extension = cache_key + '.' + output_extension
 
-            output_filename_without_extension = input_filename_without_extension[:(59 - len(output_extension))]  # Truncate filename to prevent it going over 60 chars
+            output_filename_without_extension = input_filename_without_extension[
```

Comment:
Instead of breaking the code, I think it would be neater to move the comment to the previous line here:
```python
# Truncate filename to prevent it going over 60 chars
output_filename_without_extension = input_filename_without_extension[:(59 - len(output_extension))]
```